scripts/logid-generator.py

```diff
@@ -182,17 +167,15 @@ def main(directory, check):
   max_id = prefix
```

Comment:
You will need an int conversion here, otherwise max_id becomes a "real" number which will result in new log id-s becoming something like "number.0"

In previous code, this was implicitly done by rewriting max_id which is now only done if the "prefix" is lower than the current max id which is normally not the case